meeting-notes/2018/2018-12-10--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,189 @@
+ # 💫Golang Core Dev Team Weekly Sync 🙌🏽 December 10, 2018
+
+- **Lead:** @eingenito
+- **Notetaker:**
+- **Attendees:**
+  - _attendee names..._
+  - @djdv
+  - @bigs
+  - @hannahhoward
+  - @kevina
+  - @aschmahmann
+  - @michaelavila
+  - @alanshaw
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+  - How much fun did you have working on ipfs/ipld/libp2p last week?
+- 2019 OKRs
+  - Reflection - https://docs.google.com/document/d/1gdxLWcFZPQN5921Br2kSEJqMjHx-jO5JTDEbGExMSuE/edit#
+  - New OKRs discussion - https://github.com/ipfs/team-mgmt/pull/794
+- Updates on Reflection - @eingenito
+  - *Impromptu team communication*
+  - *Tracking work priorites*
+- Updates on the big 3+ (time permitting)
+  - Data Exchange/Transfer - @hannahhoward
+  - Data Management - @magik6k
+  - Provider strategies - @michaelavila
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Is anybody looking for work?
+  - Are the high priority issues that need attention?
+
+## Notes
+
+### Week Update
+
+@name
+ - Done:
+ - Blocked:
+ - Next:
+
+@aschmahmann
+ - Done:
+   - IPNS Multi-Writer API on top of versioned graph synchronization
```

Comment:
@aschmahmann  Nice! Is there a demo that I can check out?

Reply:
The only demo currently available is the integration tests in the repo (https://github.com/aschmahmann/ipshare). I'm planning on pushing some more of these today. Would you like me to put something else together?